### DISCUSSION TITLE
Suggestions for license verification.

### DISCUSSION THREAD

**Discussion by Kim-Jiyun** [AgainIoT/Open-Set-Go#43] [Feature Suggestion / Share Ideas]
License validation refers to the process of ensuring compliance with the terms of that license before deployment. This is an important step in protecting the rights of license holders or developers and preventing piracy or unauthorized use.

I think most of the features that our team has developed so far are geared toward "starters." So I thought about putting in the license verification capabilities to not only start the project, but also to apply it to projects that already exist, or to make it manageable continuously.

Feel free to comment on what others think about this!

라이센스 검증은 배포하기 전에 해당 라이센스의 조건을 준수하는지 확인하는 과정을 의미합니다. 이는 라이센스 소유자나 개발자의 권리를 보호하고 불법 복제나 무단 사용을 방지하기 위해 중요한 단계입니다.

저희 팀이 지금까지 개발한 대부분의 기능이 "스타터"에 맞춰져 있다고 생각합니다. 그래서 라이센스 검증 기능을 넣어서 프로젝트를 시작할 때 뿐만아니라 이미 있는 프로젝트에 적용하거나, 지속적으로 관리할 수 있게 만들면 어떨까 하는 생각이 들었습니다.

이에 대해 다른 분들은 어떻게 생각하시는지 자유롭게 의견 남겨주세요!

**Comment by ymw0407** [comment]
Let's talk about this topic maintainers~
@AgainIoT/development-team

**Comment by ymw0407** [comment]
I think it's a really good idea!!! Adding that functionality seems to be the main advantage that the service is not only used for the first time to start an open source project, but also continuously available to users. Starting from this, it would be great to become an Open Source Project management service beyond just Open Source Project Starter!
정말 좋은 아이디어인 것 같습니다!!! 해당 기능을 추가하면, 서비스가 opensource 프로젝트를 시작할 처음만 사용되는게 아니라 사용자들이 지속적으로 사용할 수 있다는 점이 가장 큰 메리트인 것 같습니다. 이것을 기점으로 단순히 Open Source Project Starter을 넘어서 Open Source Project management service가 되면 좋을 것 같습니다!

**Reply by ymw0407** [reply]
Not only that, but now in the Open-Set-Go v2.0 update, I think it's good to make it work as an Open Source Starter and then go to the Open Source Project Management service!!
뿐만 아니라, 이제 Open-Set-Go v2.0 업데이트에서는 Open Source Starter의 역할을 잘 할 수 있도록 만들고 그 이후의 비전을 Open Source Project Management service로 가면 좋을 것 같습니다!!

**Comment by ymw0407** [comment]
For license verification features, we are currently considering utilizing FossLight and Olive-cli!
License 검증 기능과 같은 경우에는 현재 FossLight와 Olive-cli의 활용에 대해서 고려 중에 있습니다!

**Reply by ymw0407** [reply]
### Olive
https://olive.kakao.com/en-docs/cli/v2/overview

### Fosslight
https://fosslight.org/fosslight-guide-en/

**Comment by ymw0407** [comment]
We held a planning meeting with the relevant contents. As a result, we decided to aim to include development for that content in Open-Set-GO v2.0! Thank you for your opinion!
해당 내용으로 기획 회의를 진행하였습니다. 그 결과, Open-Set-GO v2.0에 해당 내용에 대해 개발을 포함하는 것을 목표로 하도록 결정하였습니다! 의견 감사합니다!
